crates/typst-library/src/text/raw.rs

```diff
@@ -247,7 +262,34 @@ impl Synthesize for RawElem {
 impl Show for RawElem {
     #[tracing::instrument(name = "RawElem::show", skip_all)]
     fn show(&self, _: &mut Vt, styles: StyleChain) -> SourceResult<Content> {
-        let text = self.text();
+        let mut text = self.text();
```

Comment:
could you extract this into a function `convert_tabs(text: &str, tab_size: usize) -> EcoString` and call that in the `if`? this one is getting pretty big.